pkg/debug/scripts/collect-info.sh

```diff
@@ -209,6 +209,17 @@ collect_network_info()
     echo "- done network info"
 }
 
+collect_pillar_memory_backtraces()
+{
+    echo "- pillar memory backtraces"
+
+    eve http-debug > /dev/null 2>&1
+    eve exec debug curl --retry-all-errors --retry 5 -m 5 "http://127.1:6543/debug/pprof/heap?debug=1" > "$DIR/pillar-memory-backtraces"
```

Comment:
Does this wait between each try?

Reply:
Yes it does and it increases the waiting time.

Reply:
f.e.:
```
christoph@ponyhof ~/p/b/b/buildXXX (keep_aot_bytecode)> curl --retry-all-errors --retry 5 127.1:1
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
Warning: Problem (retrying all errors). Will retry in 1 seconds. 5 retries 
Warning: left.
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
Warning: Problem (retrying all errors). Will retry in 2 seconds. 4 retries 
Warning: left.
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
Warning: Problem (retrying all errors). Will retry in 4 seconds. 3 retries 
Warning: left.
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
Warning: Problem (retrying all errors). Will retry in 8 seconds. 2 retries 
Warning: left.
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
Warning: Problem (retrying all errors). Will retry in 16 seconds. 1 retries 
Warning: left.
curl: (7) Failed to connect to 127.0.0.1 port 1 after 0 ms: Couldn't connect to server
```

Reply:
Nice